assets/xml/objects/object_oyu.xml

```diff
@@ -1,9 +1,10 @@
 ﻿<Root>
+    <!-- Hot Spring Water found in the Woodfall region and the Empty Ikana Well -->
     <File Name="object_oyu" Segment="6">
-        <DList Name="object_oyu_DL_000080" Offset="0x80" />
-        <DList Name="object_oyu_DL_000158" Offset="0x158" />
-        <Texture Name="object_oyu_Tex_000160" OutName="tex_000160" Format="rgba16" Width="32" Height="32" Offset="0x160" />
-        <TextureAnimation Name="object_oyu_Matanimheader_000968" Offset="0x968" />
-        <Collision Name="object_oyu_Colheader_000988" Offset="0x988" />
+        <DList Name="gSpringWaterDL" Offset="0x80" />
```

Comment:
Given that there is both Spring water and Hot spring water, differentiating them is important. So this needs to be `gHotSpringWaterDL`

Reply:
I'll make sure to change that, I didn't really know how the normal water was named so I just thought of it as "Water"